packages/react-components/react-switch/src/stories/SwitchBestPractices.md

```diff
@@ -1,5 +1,15 @@
-## Best practices
+<details>
+<summary>
+ Best Practices
+</summary>
 
-### Do
+### Layout
 
-### Don't
+- When people need to perform extra steps for changes to take effect, use a check box instead. For example, if they must click a "Submit", "Next", or "OK" button to apply changes, use a check box.
+
+### Content
+
+- Only replace the On/Off labels if there are more specific labels for the setting. For example, you might use Show/Hide if the setting is "Show images".
+- Keep descriptive text short and concise—two to four words; preferably nouns. For example, "Focused inbox" or "WiFi".
+
+</details>
```

Comment:
Added best practices & component description, took it from existing docs (https://developer.microsoft.com/en-us/fluentui#/controls/web/toggle). We will probably need to follow up on it to cleanup, but it's better than we have currently:

![image](https://user-images.githubusercontent.com/14183168/170232004-7049d1f2-e047-4c20-9526-54e721e8bfd6.png)